mobile-browser-based-version/server/src/logic/federated/async_weights_buffer.ts

```diff
@@ -22,6 +23,7 @@ export class AsyncWeightsBuffer {
     buffer: Map<string, any>;
     round: number;
     roundCutoff: number;
+    observers: AsyncWeightsInformant[]
```

Comment:
Why would there be a list of weight informants rather than just one?

Reply:
Good question, I'll remove that